backend/src/main/java/wooteco/prolog/roadmap/application/QuizService.java

```diff
@@ -32,9 +35,20 @@ public Long createQuiz(Long keywordId, QuizRequest quizRequest) {
         return quiz.getId();
     }
 
-    public QuizzesResponse findQuizzesByKeywordId(Long keywordId) {
+    public QuizzesResponse findQuizzesByKeywordId(Long keywordId, Long memberId) {
         final List<Quiz> quizzes = quizRepository.findFetchQuizByKeywordId(keywordId);
-        return QuizzesResponse.of(keywordId, quizzes);
+        final List<QuizResponse> quizResponses = quizzes.stream()
+            .map(quiz -> QuizResponse.of(quiz, isLearning(memberId, quiz.getId())))
+            .collect(Collectors.toList());
+        return QuizzesResponse.of(keywordId, quizResponses);
+    }
+
+    private boolean isLearning(Long memberId, Long quizId) {
+        if (memberId == null) {
```

Comment:
```suggestion
if (isNull(memberId)) {
```

개인 취향이지만 Objects의 정적 메서드를 쓰면 예쁠 것 같아요!

Reply:
반영했습니다~